master/getting-started/kubernetes/installation/hosted/k8s-backend/index.md

```diff
@@ -32,14 +34,27 @@ Once installed, you can try out NetworkPolicy by following the [simple policy gu
 
 Below are a few examples for how to get started.
 
+### Using Kubernetes' addon-manager
+
+Kubernetes' [addon-manager](https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/addon-manager)
+can be used for deploying Calico alongside with other cluster addons. It takes
```

Comment:
how bout

The addon-manager monitors `/etc/kubernetes/addons` for YAML manifests and ensures they are applied to the cluster.